server/events/vcs/github_client_test.go

```diff
@@ -317,6 +319,7 @@ func TestGithubClient_HideOldComments(t *testing.T) {
 	Ok(t, err)
 	Equals(t, 3, len(gotMinimizeCalls))
 	Equals(t, "6", gotMinimizeCalls[0].Variables.Input.SubjectID)
+	Equals(t, "8", gotMinimizeCalls[1].Variables.Input.SubjectID)
 	Equals(t, "9", gotMinimizeCalls[2].Variables.Input.SubjectID)
```

Comment:
Seems like the initial tests wanted to cover this use case but were bogus since `Continued Plan from previous comment` is not in any of the comment strings?